akka-http-server/src/main/scala/com/avast/sst/akka/http/server/AkkaHttpActorSystemModule.scala

```diff
@@ -0,0 +1,45 @@
+package com.avast.sst.akka.http.server
+
+import akka.actor.ActorSystem
+import cats.effect.{Resource, Sync}
+import com.typesafe.config.{Config, ConfigFactory}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.language.higherKinds
+import scala.util.Try
+
+// This is only temporarily part of this module

```

Comment:
I can split it into separate module, but I don't plan to invest much more in that. It is a daunting task as `ActorSystem` configuration is humongous.

Reply:
I understand. Why not get rid of this completely if we're not going to support it properly? It will force everyone to implement this but it's just fair if we're not gonna do that. I think that would be the best approach.